### PR TITLE
feat:Added automation for checking PR Title

### LINKED ISSUE #94: Add some automation around checking commit style
**Problem statement.**
Some of the contributors to datadog-service haven't contributed to Keptn before so they usually don't know the [commit](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#commit-types-and-scopes) and the [PR style](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#make-a-pull-request) we follow. We have to tell them about this when they make their first PR. 

**Describe the solution you'd like**
It'd be nice to have an automated check for commit and PR style. It could be a github workflow on PRs. The check should also suggest the right way to do things (i.e., how to fix the current commit or PR)

**Additional context**
We have a [similar check in Keptn](https://github.com/keptn/keptn/actions/runs/3171957555/jobs/5165919432) for PR titles (we use [this](https://github.com/amannn/action-semantic-pull-request) github action for that)

**TODO**
- [ ] Add PR title check
- [ ] Add PR commit check
- [ ] Make sure they work (screenshots or something else which makes it easier to review)
- [ ] Raise a PR and get it reviewed
- DONE

### LINKED ISSUE #121: Automated check for PR Title
Since new contributors who has not contributed to keptn previously do not know the PR title style keptn follows.
This is to automate the PR title check whenever a new PR is created.

### PR DESCRIPTION
**Pull request by tarangverma**
This will help beginners to follow the commit style which Keptn is following.
closes #121 
Signed-off-by: Tarang Verma <tarangverma004@gmail.com>

### CODE REVIEW

**Comment by tarangverma** [comment]
Screenshot (22)
It can be seen here..

**Comment by vadasambar** [comment]
> This is a [GitHub Action](https://github.com/features/actions) that ensures your PR title matches the [Conventional Commits spec](https://www.conventionalcommits.org/).

https://github.com/amannn/action-semantic-pull-request#action-semantic-pull-request

@tarangverma this PR adds support for checking the PR title which is a good thing to have but it won't fix https://github.com/keptn-sandbox/datadog-service/issues/94 . You can create a separate Github issue for checking PR title and we can say this PR fixes that issue.

**Comment by tarangverma** [comment]
i think am done with the changes..

File: .github/workflows/validate-semantic-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Semantic PR Validation
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```
Comment (vadasambar) [inline .github/workflows/validate-semantic-pr.yaml:7]: :memo: 
> synchronize: Triggered when a pull request's head branch is updated. For example, when the head branch is updated from the base branch, when new commits are pushed to the head branch, or when the base branch is changed.

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request

**Comment by vadasambar** [comment]
And merged!

**Pull request merged**